src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -492,6 +492,7 @@ public unsafe Vector2i Size
         /// </summary>
         /// <remarks>
         /// Pass <c>-1</c> as the width or height to ignore the respective dimension.
+        /// If you set size limits and an aspect ratio that conflict, the results are undefined.
```

Comment:
I would say `aspect ratio lock` instead of just `aspect ratio` as I think it's somewhat unclear otherwise.